components/ILIAS/Mail/classes/Preview/class.ilCourseMailTemplateTutorContextPreview.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+class ilCourseMailTemplateTutorContextPreview extends ilCourseMailTemplateTutorContext
```

Comment:
Shouldn't this class be moved to the `Course` component?

Furthermore, I suggest removing obsolete docs (without any added value) and introducing a blank line between the `case` statements of the `swtich` in `resolveSpecificPlaceholder`.